### PR TITLE
Bugfix ipa320/cob_robots#227

### PR DESCRIPTION
**Pull request by ghost**
bugfix ipa320/cob_robots#227
problem description: robot returns success in initialization of the platform although emergency button is active and thus physically prevents performing homing procedure (motors signal communication error);
to fix that bug (in ipa320/cob_command_tools/cob_teleop): now checking success of init/recover procedures instead of just checking success of service calls;
to fix that bug (in ipa320/cob_driver/base_drive_chain): changed wrong ROS output regarding base recover service callback

### CODE REVIEW

File: cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp [hunk 1]
```diff
@@ -515,11 +515,10 @@ class NodeClass
 			}
 			else
 			{
-				ROS_WARN("...base already recovered...");
-				res.success.data = true;
-				res.error_message.data = "base already recovered";
+				ROS_WARN("Base not yet initialized.");
+				res.success.data = false;
```
Comment (floweisshardt) [inline cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp:519]: why did you change the success behaviour? 
The `res.success.data` should contain the state of the base which means `true` if the base can be moved afterwards (recover successfully) and `false` if the base cannot be moved (recover failed).

Comment (floweisshardt) [inline cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp:519]: sorry, my mistake. looks good.

**Comment by floweisshardt** [comment]
visual review: OK

File: cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp [hunk 2]
```diff
@@ -430,7 +430,7 @@ class NodeClass
 		bool srvCallback_Init(cob_srvs::Trigger::Request &req,
 							  cob_srvs::Trigger::Response &res )
 		{
-			ROS_DEBUG("Service Callback init");
+			ROS_DEBUG("Service callback /base_controller/init");
```
Comment (fmessmer) [inline cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp:433]: @ipa-mig-tk @ipa-mig 
even if this is only ROS_DEBUG, can we get rid of the hardcoded namespace `/base_controller` in the text - as this is going to changed anyway.
Without the hardcoded namespace we are old-new-namespace-independent...

File: cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp [hunk 3]
```diff
@@ -500,7 +500,7 @@ class NodeClass
 		{
 			if(m_bisInitialized)
 			{
-				ROS_DEBUG("Service callback reset");
+				ROS_DEBUG("Service callback /base_controller/recover");
```
Comment (fmessmer) [inline cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp:503]: @ipa-mig-tk @ipa-mig 
even if this is only ROS_DEBUG, can we get rid of the hardcoded namespace `/base_controller` in the text - as this is going to changed anyway.
Without the hardcoded namespace we are old-new-namespace-independent...

File: cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp (hunk 2 above)
Comment (mgruhler) [inline cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp:433]: @ipa-mig-tk Please take care of this. ipa-fxm has a point there. Didn't think of it when talking to you...
Just call it "Service init in cob_base_drive_chain" or something (same for below)

**Comment by ghost** [comment]
Done. Sending a PR.
